roles/clevis/tasks/install-RedHat.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Install clevis packages
+  become: true
+  ansible.builtin.dnf:
```

Comment:
If you could use the generic ansible.builtin.package, this also could be generic instead of distro-specific.

Reply:
in the command before i try to explain it . The packages in EL look like this:  clevis-15-1.el8.x86_64